src/copentimelineio/anyDictionary.cpp

```diff
@@ -58,15 +58,15 @@ OTIO_API AnyDictionaryIterator* AnyDictionary_erase_range(
 }
 OTIO_API int AnyDictionary_erase_key(AnyDictionary* self, const char* key)
```

Comment:
tbh i think we should revisit this part of the api. erase in the C++ side returns an iterator. on this side of the fence, i'm not sure an int is meaningful, or at least I'm not spotting where we could use the int